SPECS/hiredis/hiredis.spec

```diff
@@ -0,0 +1,193 @@
+Summary:        Minimalistic C client library for Redis
+Name:           hiredis
+Version:        1.0.2
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/redis/hiredis
+Source0:        https://github.com/redis/hiredis/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  openssl-devel
+BuildRequires:  redis
+
+%description
+Hiredis is a minimalistic C client library for the Redis database.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+This package contains libraries and header files for
+developing applications that use %{name}.
+
+%prep
+%autosetup -p1
+
+%build
+%make_build PREFIX="%{_prefix}" LIBRARY_PATH="%{_lib}"     \
+            DEBUG="%{optflags}" LDFLAGS="%{?__global_ldflags}" USE_SSL=1
+
+%install
+%make_install PREFIX="%{_prefix}" LIBRARY_PATH="%{_lib}"
+
+find %{buildroot} -name '*.a' -delete -print
+
+%check
+# TODO: Koji isolated environment may cause some tests fail to pass.
```

Comment:
Let's drop the koji comment and also drop the or condition in the test.

Reply:
Done @mfrw